app-editors/gnome-text-editor/gnome-text-editor-43.0.ebuild

```diff
@@ -0,0 +1,58 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit gnome.org gnome2-utils meson xdg
+
+DESCRIPTION="A simple text editor for the GNOME desktop"
+HOMEPAGE="https://gitlab.gnome.org/GNOME/gnome-text-editor"
+SRC_URI="mirror://gnome/sources/gnome-text-editor/42/gnome-text-editor-${PV/_/.}.tar.xz"
```

Comment:
Just drop `SRC_URI=...`. It's wrong, and I'm not sure why it's there in the v42 ebuild.

````
>>> Downloading 'https://download.gnome.org/sources/gnome-text-editor/42/gnome-text-editor-43.0.tar.xz'
--2022-09-21 09:00:00--  https://download.gnome.org/sources/gnome-text-editor/42/gnome-text-editor-43.0.tar.xz
Resolving download.gnome.org (download.gnome.org)... 8.43.85.3, 8.43.85.4, 8.43.85.5
Connecting to download.gnome.org (download.gnome.org)|8.43.85.3|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2022-09-21 09:00:00 ERROR 404: Not Found.

!!! Couldn't download 'gnome-text-editor-43.0.tar.xz'. Aborting.
````

Reply:
I couldn't get it to work without the SRC_URI, I figured that upstream does something with where they put the file that is different from what the other gnome people do.